src/document_parser_callbacks.h

```diff
@@ -27,37 +27,40 @@ really_inline error_code parser::on_success(error_code success_code) noexcept {
 }
 really_inline bool parser::on_start_document(uint32_t depth) noexcept {
   containing_scope_offset[depth] = current_loc;
+  containing_scope_count[depth] = 0;
```

Comment:
Is this just out of an abundance of caution (which would be fine)? Or are we doing something with count in the document's tape entry?

Reply:
This line could be dropped.

Reply:
I think I'd leave it, it'll make people nervous otherwise and it (ought to be) not a performance impacter.